draft-bbs-signatures.md

```diff
@@ -1082,13 +1109,13 @@ The parameters that each ciphersuite needs to define are generally divided into
 
 - hash: a cryptographic hash function.
 
-- xof\_no\_of\_bytes: Number of bytes to draw from the xof when performing operations such as creating generators as per the operation documented in (#generator-point-computation) or computing the e and s components of the signature generated in (#sign). It is RECOMMENDED this value be set to one greater than `ceil(r+k)/8` for the ciphersuite, where `r` and `k` are parameters from the underlying pairing friendly curve being used.
+- expand\_length: Number of bytes to expand a message that will be hashed to a scalar (see [Hashing to Scalars](#hashing-to-scalars)) or to a point in G1 (i.e., a generator, see [Create Generators](#creategenerators)). It is RECOMMENDED that this value is set to one greater or equal to `ceil(r+k)/8` for the ciphersuite, where `r` and `k` are parameters from the underlying pairing friendly curve being used.
```

Comment:
I don't think `ceil(r+k)/8` is correct (it's huge). Maybe `ceil((ceil(log2(p)) + k) / 8)`

Reply:
Good catch! Yea it was wrong. Fixed.